internal/schemas.ts

```diff
@@ -790,6 +790,36 @@ const CompressedImage: FoxgloveMessageSchema = {
   ],
 };
 
+const CompressedVideo: FoxgloveMessageSchema = {
+  type: "message",
+  name: "CompressedVideo",
+  description: "A single frame of a compressed video bitstream",
+  fields: [
+    {
+      name: "timestamp",
+      type: { type: "primitive", name: "time" },
+      description: "Timestamp of image",
+    },
+    {
+      name: "frame_id",
+      type: { type: "primitive", name: "string" },
+      description:
+        "Frame of reference for the image. The origin of the frame is the optical center of the camera. +x points to the right in the image, +y points down, and +z points into the plane of the image.",
+    },
+    {
+      name: "data",
+      type: { type: "primitive", name: "bytes" },
+      description:
+        "Compressed video frame data. For packet-based video codecs this data must begin and end on packet boundaries (no partial packets), and must contain enough video packets to decode exactly one image (either a keyframe or delta frame).",
+    },
+    {
+      name: "format",
+      type: { type: "primitive", name: "string" },
+      description: "Video format\n\nSupported values: `h264`",
+    },
+  ],
+};
+
```

Comment:
Why are `keyframe` and `metadata` no longer needed here?

Reply:
these are no longer included because people often come from already having compressed video formatted streams and it would be too much of a heavylift for them to parse out this format-specific information and write it back to the message in a more accessible way. All of this information is included in the formatted `data` byte array